django/webscraper/models.py

```diff
@@ -0,0 +1,77 @@
+"""This module defines models (and useful functions connected to them),
+for webscraper application"""
+import datetime
+from django.db import models
+
+
+# model was previously in pieskiUW.models, but models shouldn't
+# be kept in a project directory. Read more:
+# https://stackoverflow.com/questions/2610727/django-project-models-py-versus-app-models-py#:~:text=Django%20models%20can%20only%20reside%20in%20applications%2C%20not,in%20the%20places%20where%20it%20would%20make%20sense.
+# Credit for creating model to jpalikowska
+class Pet(models.Model):
+    """Model of pet table"""
+
+    SEX = (
+        ("m", "male"),
+        ("f", "female"),
+    )
+    no = models.CharField(max_length=20)
+    name = models.CharField(max_length=100)
+    breed = models.CharField(max_length=100)
+    age = models.IntegerField()
+    gender = models.CharField(max_length=1, choices=SEX)
+    weight = models.IntegerField()
+    status = models.CharField(max_length=50)
+    date_in = models.DateField()
+    date_out = models.DateField()
+    address_found = models.CharField(max_length=100)
+    box = models.CharField(max_length=50)
+    group_name = models.CharField(max_length=50)
+    group_link = models.URLField()
+    link = models.URLField()
+
+
+class WebscrappingProcess(models.Model):
+    """Model of process doing the webscrapping"""
+
+    pid = models.IntegerField()
+    working = models.BooleanField()
+    timestamp = models.DateTimeField(auto_now_add=True)
+
+
+# delete all WebscrappingProcess objects
+def clear_webscrapping_processes():
+    """Makes sure there are no redundant PIDs stored in database"""
```

Comment:
```suggestion
    """Removes all PIDs stored in database"""
```

Reply:
I cannot see any benefit in this change.